system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -911,8 +911,9 @@ component {
 		for( var i=1; i <= formItems.len(); i++ ) {
 			if ( formItems[i].type.isFormField ) {
 				var columns = renderingService.getItemTypeExportColumns( formItems[i].type.id, formItems[i].configuration );
+				var export  = formItems[i].configuration.export ?: 0 ;
```

Comment:
I suggest this should be more defensive. The following approach is solid for ensuring boolean values:

```
var export = IsBoolean( formItems[ i ].configuration.export ?: "" ) && formItems[ i ].configuration.export;
```

Reply:
Also, only get the columns if `export = true`, so switch lines 913 + 914 around and do:

```
var columns = export ? renderingService.getItemTypeExportColumns( formItems[i].type.id, formItems[i].configuration ) : [];
```

Reply:
The changes requested had been pushed.